### ISSUE TITLE
Loot is not beeing dropped when Husksync is installed.

### ISSUE THREAD

**Issue by xPoyraz** [SirBlobman/CombatLogX#775]
### Server Implementation

Paper

### Server Version

other

### Describe the bug

If a player dies by leaving the game during a battle with a combatlog tag, their items do not fall to the ground, this problem disappears when husksync is not installed.

This problem occurs even though I am not using HuskSync inventory synchronisation

### Reproduction Steps

1. Run a server with Combatlog and husksync
2. players exit the game during battle
3. and items do not fall to the ground

### Expected behaviour

items must fall to the ground when the player dies leaving the game

### CombatLogX Version

11.4.0.2.Beta-1222

**Comment by SirBlobman** [comment]
Please send the full output of the `/clx version` command.

**Comment by xPoyraz** [comment]
> Please send the full output of the `/clx version` command.

[02:35:12 INFO]: Java Version: 17.0.9
[02:35:12 INFO]: Java Vendor: Oracle Corporation
[02:35:12 INFO]: Java URL: N/A
[02:35:12 INFO]: Server Version: git-Purpur-2095 (MC: 1.20.2)
[02:35:12 INFO]: Bukkit Version: 1.20.2-R0.1-SNAPSHOT
[02:35:12 INFO]: Minecraft Version: 1.20.2
[02:35:12 INFO]: NMS Version: 1_20_R2
[02:35:12 INFO]: Dependency Information:
[02:35:12 INFO]:  - AngelChest (not installed)
[02:35:12 INFO]:  - ASkyBlock (not installed)
[02:35:12 INFO]:  - BentoBox (not installed)
[02:35:12 INFO]:  - Citizens (not installed)
[02:35:12 INFO]:  - CMI v9.6.9.3
[02:35:12 INFO]:  - CrackShot (not installed)
[02:35:12 INFO]:  - CrashClaim (not installed)
[02:35:12 INFO]:  - Essentials (not installed)
[02:35:12 INFO]:  - FabledSkyBlock (not installed)
[02:35:12 INFO]:  - FeatherBoard (not installed)
[02:35:12 INFO]:  - FlagWar (not installed)
[02:35:12 INFO]:  - GriefDefender (not installed)
[02:35:12 INFO]:  - GriefPrevention (not installed)
[02:35:12 INFO]:  - HuskHomes v4.5.3
[02:35:12 INFO]:  - HuskSync v3.2
[02:35:12 INFO]:  - HuskTowns (not installed)
[02:35:12 INFO]:  - iDisguise (not installed)
[02:35:12 INFO]:  - IridiumSkyblock (not installed)
[02:35:12 INFO]:  - Kingdoms (not installed)
[02:35:12 INFO]:  - Konquest (not installed)
[02:35:12 INFO]:  - Lands (not installed)
[02:35:12 INFO]:  - LibsDisguises (not installed)
[02:35:12 INFO]:  - MarriageMaster (not installed)
[02:35:12 INFO]:  - MCPets (not installed)
[02:35:12 INFO]:  - MythicMobs (not installed)
[02:35:12 INFO]:  - PlaceholderAPI v2.11.5
[02:35:12 INFO]:  - PlayerParticles (not installed)
[02:35:12 INFO]:  - PreciousStones (not installed)
[02:35:12 INFO]:  - PremiumVanish (not installed)
[02:35:12 INFO]:  - ProtectionStones (not installed)
[02:35:12 INFO]:  - ProtocolLib v5.1.1-SNAPSHOT-673
[02:35:12 INFO]:  - RedProtect (not installed)
[02:35:12 INFO]:  - Residence (not installed)
[02:35:12 INFO]:  - Sentinel (not installed)
[02:35:12 INFO]:  - SuperiorSkyblock2 (not installed)
[02:35:12 INFO]:  - SuperVanish (not installed)
[02:35:12 INFO]:  - Towny (not installed)
[02:35:12 INFO]:  - UltimateClaims (not installed)
[02:35:12 INFO]:  - uSkyBlock (not installed)
[02:35:12 INFO]:  - VanishNoPacket (not installed)
[02:35:12 INFO]:  - WorldGuard (not installed)
[02:35:12 INFO]:  - ViaVersion v4.9.2
[02:35:12 INFO]:  - BlueSlimeCore v2.9.4.Beta-391
[02:35:12 INFO]: CombatLogX by SirBlobman
[02:35:12 INFO]: Local Version: 11.4.0.2.Beta-1222
[02:35:12 INFO]: Remote Version: Not Available
[02:35:12 INFO]: Enabled Expansions (11):
[02:35:12 INFO]:  - Action Bar v17.3
[02:35:12 INFO]:  - Boss Bar v17.1
[02:35:12 INFO]:  - Cheat Prevention v17.6
[02:35:12 INFO]:  - HuskHomes Compatibility v17.2
[02:35:12 INFO]:  - LuckPerms Compatibility v17.1
[02:35:12 INFO]:  - PlaceholderAPI Compatibility v17.2
[02:35:12 INFO]:  - Damage Effects v17.0
[02:35:12 INFO]:  - Damage Tagger v17.1
[02:35:12 INFO]:  - Death Effects v17.0
[02:35:12 INFO]:  - End Crystal Helper v17.1
[02:35:12 INFO]:  - Newbie Helper v17.2


"HuskSync Compatibility" why doesn't it show up even though it's installed?

**Comment by SirBlobman** [comment]
Thank you, I'll look into this once I'm able to make a proper test server for HuskSync.

**Comment by xPoyraz** [comment]
Meanwhile, the exiting player's items are deleted but not dropped, something destroys the exiting player's items instead of dropping them. 
When I delete HuskSync, it fixs

**Comment by umutc2n** [comment]
I have the same problem; I use HuskSync only to save stats, all other settings are turned off, I do not use HuskSync Expansions in combatlogx, but still, when husksync and combatlogx are installed at the same time, if the player leaves the game during the battle, the items do not fall to the ground and the items are deleted from the inventory of the player who leaves the game, this is an important problem.

**Comment by Olivoz** [comment]
Yeah the problem the expansion is trying to solve is HuskSync clearing the inventory of the quitting player before they are killed

**Comment by xPoyraz** [comment]
> Having the same issue. Is there any fix for this yet?

The problem still persists

**Comment by JorisvD01** [comment]
> > Having the same issue. Is there any fix for this yet?
> 
> The problem still persists 

I've found a solution that worked for me. We changed the listening priority in husksync, so it would change the order of handeling the deaths and quits. I used a few different configurations before I found a working one for us, so you might have to fiddle around a little bit with those settings.

**Comment by nathanpaulyoung** [comment]
Updating this for the next person. Setting HuskSync's Death Listener Priority to LOWEST fixes this.
